paasta_tools/cli/cmds/rerun.py

```diff
@@ -85,6 +88,21 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    rerun_parser.add_argument(
+        '-t', '--rerun-type',
+        dest="rerun_type",
+        choices=['instance', 'graph'],
+        help="Specify how to rerun jobs that have parent-dependencies.\n"
+             "  - instance: rerun, as soon as possible, the required instance ONLY\n"
+             "  - graph: will rerun, as soon as possible, ALL the instances related to the required instance.\n"
+             "    NOTE: it could be expensive in terms of resources and of time. Use it carefully.\n"
+             "\n"
+             "Example: Assume that we have 4 jobs (j1, j2, j3 and j4) with the following relations\n"
+             "    j1 -> j2, j1 -> j3, j2 -> j3, j2 -> j4\n"
+             "\n"
+             "    Rerunning j2 wih --rerun-type=instance will rerun ONLY j2, j3 and j4 will not be re-ran\n"
+             "    Rerunning j2 wih --rerun-type=graph will rerun j1, j2, j3 and j4\n"
```

Comment:
Can you make it clear in the help text that it will follow the correct dependent graph ordering?